SUPPORTEDTECH.rst

```diff
@@ -0,0 +1,111 @@
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Non Equal Comparison                    |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Less or Equal Comparison                |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Greater or Equal Comparison             |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Less than Comparison                    |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Greater than Comparison                 |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Mult, matmul, conv2d, conv2d\_transpose |                     2 |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | Addition                                |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | Substraction                            |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | Division                                |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | xor                                     |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | Equal Comparison                        |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | Less than Comparison                    |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | Greater than Comparison                 |                    2+ |
```

Comment:
Also add non-linear operations like tanh, relu, sigmoid, maxpool and few others I sent you. (Also you can find it in approximations and functional module).

Reply:
I know, but... Are all of them supported by all the protocols except ABY3? How can I see that in the code?

Reply:
So we don't use ABY3 directly, but other protocols use ABY3. For example, if a user wants to train a Linear regression model they cannot train it using ABY3. But, they can train it using Falcon which uses ABY3. If someone else wants to write their own protocol extending SyMPC, they know that it has the functionality of ABY3 they can use.